readme.md

```diff
@@ -75,6 +75,7 @@ Check out my [blog](https://blog.sindresorhus.com) and follow me on [Twitter](ht
 	- [Must Watch Talks](https://github.com/bolshchikov/js-must-watch)
 	- [Tips](https://github.com/loverajoel/jstips)
 	- [Network Layer](https://github.com/Kikobeats/awesome-network-js)
+	- [Micro NPM Modules](https://github.com/parro-it/awesome-microlibs)
```

Comment:
Your list title is also very vague, `microlibs`. Could be any language/ecosystem.